lib/model/config.dart

```diff
@@ -182,12 +189,14 @@ class Config {
     required this.navigatorKey,
     this.origin,
     this.customParameters = const {},
+    required String postLogoutRedirectUri,
```

Comment:
@Musaddiq625 I think this parameter should be optional

Reply:
hm.. okay, I understand